modules/validations/oneway_onelane.js

```diff
@@ -0,0 +1,71 @@
+import { actionChangeTags } from '../actions/change_tags';
+import { t } from '../core/localizer';
+import { validationIssue, validationIssueFix } from '../core/validation';
+import { utilDisplayLabel } from '../util';
+
+
+export function validationOneLaneWithNoOneway() {
+  const type = 'oneway_onelane';
+
+  /** @param {string} onewayValue  @param {osmEntity} entity */
+  function createSuggestion(onewayValue, entity) {
+    return new validationIssueFix({
+      icon: 'iD-icon-data',
+      title: t.append(`issues.fix.tag_as_oneway_${onewayValue}.title`),
+      onClick: (context) => {
+        const newTags = { ...entity.tags, oneway: onewayValue };
+        context.perform(
+          actionChangeTags(entity.id, newTags), t(`issues.fix.tag_as_oneway_${onewayValue}.annotation`)
+        );
+      }
+    });
+  }
+
+  /** @param {osmEntity} entity */
+  function makeOneLaneIssue(entity) {
+    return new validationIssue({
+      type,
+      subtype: type,
+      severity: 'warning',
+      message: (context) => t.append(
+        'issues.oneway_onelane.message',
+        { feature: utilDisplayLabel(entity, context.graph()) }
+      ),
+      reference: showReference,
+      entityIds: [entity.id],
+      hash: type,
+      dynamicFixes: () => [
+        // suggest adding `oneway` tag (automatic)
+        ...['yes', 'alternating'].map(onewayValue => createSuggestion(onewayValue, entity)),
+
+        // or changing the `lanes` tag (manual)
+        new validationIssueFix({ title: t.append('issues.fix.change_lane_tag.title') })
+      ]
+    });
+  }
+
+
+  function showReference(selection) {
+    selection.selectAll('.issue-reference')
+      .data([0])
+      .enter()
+      .append('div')
+      .attr('class', 'issue-reference')
+      .call(t.append(`issues.${type}.reference`));
+  }
+
+  /** @param {osmEntity} entity */
+  const validation = (entity) => {
+    const isOneLane = entity.tags.lanes === '1';
+
+    if (entity.type !== 'way' || entity.isOneWay() || !isOneLane) return [];
+
+    // this line has lanes=1, but is not oneway. This makes no sense.
```

Comment:
Just to make sure we’re covering all our bases:

* One-lane bridges are very common in some rural areas where I map. Traffic can enter from either direction, but not at the same time. Technically the lane is reversible/alternating, but not on a schedule and not controlled by a stoplight as illustrated on the `oneway=alternating` page. Would this situation still be `oneway=alternating`?
* Are there any situations where we’d need to consider conditional tags?

Reply:
My first thought was also one-lane bridges - they are very common.

Reply:
> - [...] Technically the lane is reversible/alternating, but not on a schedule and not controlled by a stoplight as illustrated on the `oneway=alternating` page. Would this situation still be `oneway=alternating`?

My understanding is that this is a perfect example of `oneway=alternating`. If there are no traffic lights then the [`priority`](https://osm.wiki/Key:priority) tag is useful, which [the wiki page](https://osm.wiki/Tag:oneway=alternating) mentions.

> - Are there any situations where we’d need to consider conditional tags?

I can't think of any situatations because in this case, `oneway:conditional` should be combined with a valid default `oneway` tag (`yes`/`-1`/`alternating`/`reversible`), like [this example from Brisbane](https://goo.gl/maps/o9TDeoLW2YD6Y9fG6)